queries.sql

```diff
@@ -85,4 +85,88 @@ FROM owners
 JOIN animals ON owners.id = animals.owner_id
 GROUP BY owners.full_name
 ORDER BY COUNT(animals.id) DESC
-LIMIT 1;
\ No newline at end of file
+LIMIT 1;
+
+--Who was the last animal seen by William Tatcher--
+SELECT animals.name 
+FROM visits 
+JOIN vets ON visits.vets_id = vets.id 
+JOIN animals ON visits.animals_id = animals.id 
+WHERE vets.name = 'William Tatcher' 
+ORDER BY date_of_visit DESC 
+LIMIT 1;
+
+
+--How many different animals did Stephanie Mendez see?--
+SELECT COUNT(animals.name) 
```

Comment:
- [ ] According to the project requirements. Add a screenshot of the query results to the pull request description. This also helps to ensure that you run the queries with success. Kindly make this 💯 👍🏻